src/config/webpack.config.js

```diff
@@ -36,26 +36,78 @@ const base = (env, argv) => {
     module: {
       rules: [
         {
-          test: /\.js$/,
-          use: {
-            loader: require.resolve('babel-loader'),
-            options: {
-              presets: [require('./babelrc')],
-              babelrc: false,
-              cacheDirectory: true
+          oneOf: [
+            {
+              test: /\.js$/,
+              include: fromRoot(paths.src),
+              use: {
+                loader: require.resolve('babel-loader'),
+                options: {
+                  presets: [require('./babelrc')],
+                  babelrc: false,
+                  cacheDirectory: true
+                }
+              }
+            },
+            {
+              test: /\.js$/,
+              exclude: /@babel(?:\/|\\{1,2})runtime/,
+              use: {
+                loader: require.resolve('babel-loader'),
+                options: {
+                  presets: [require('./babelrc')],
+                  babelrc: false,
+                  cacheDirectory: true,
+                  sourceMaps: false
+                }
+              }
             }
-          }
+          ]
         }
       ]
     },
-
     resolve: {
       alias: {
         '@babel/runtime': path.dirname(
           require.resolve('@babel/runtime/package.json')
         )
       }
     },
+    optimization: {
+      minimize: isProduction,
+      minimizer: [
+        // This is only used in production mode
+        new TerserPlugin({
+          terserOptions: {
+            parse: {
+              // we want terser to parse ecma 8 code. However, we don't want it
+              // to apply any minfication steps that turns valid ecma 5 code
+              // into invalid ecma 5 code. This is why the 'compress' and 'output'
+              // sections only apply transformations that are ecma 5 safe
+              // https://github.com/facebook/create-react-app/pull/4234
+              ecma: 8
+            },
+            compress: {
+              ecma: 5,
+              warnings: false
+            },
+            mangle: {
+              safari10: true
+            },
+            output: {
+              ecma: 5,
```

Comment:
Does this mean it is outputting ecma5 code?

Reply:
No this build outputs according to our default or repo browserlist support config !! 
This setting means that any transformation applied by terser needs to be at least es5 compatible. Makes sense ? ES6+ code will not be converted to ES5 by terser but any change in the code made by terser needs to be es5 compatible.